src/main/java/net/mcreator/ui/wysiwyg/WYSIWYGEditor.java

```diff
@@ -45,11 +45,24 @@
 import java.awt.*;
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
+import java.lang.reflect.InvocationTargetException;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class WYSIWYGEditor extends JPanel {
 
+	public static final List<WYSIWYGComponentRegistration<?>> COMPONENT_REGISTRY = new ArrayList<>() {{

```

Comment:
Is this intended to provide support for e.g. unregistering all the other component types (`COMPONENT_REGISTRY.clear()`)?

Reply:
ModElementTypeLoader has it public too, I don't see a problem in this, maybe some plugin wants to replace something